mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/ui/KYCLevel1Screen.kt

```diff
@@ -0,0 +1,239 @@
+import com.togitech.ccp.component.TogiCountryCodePicker
+import org.mifos.mobilewallet.mifospay.R
+import org.mifos.mobilewallet.mifospay.kyc.presenter.KYCLevel1UiState
+import org.mifos.mobilewallet.mifospay.kyc.presenter.KYCLevel1ViewModel
+import org.mifos.mobilewallet.mifospay.theme.MifosTheme
+import java.time.format.DateTimeFormatter
+
+
+@Composable
+fun KYCLevel1Screen(
+    viewModel: KYCLevel1ViewModel = hiltViewModel(),
+    navigateBack: () -> Unit
+) {
+    val context = LocalContext.current
+    val kyc1uiState by viewModel.kyc1uiState.collectAsStateWithLifecycle()
+
+    KYCLevel1Screen(
+        context = context,
+        uiState = kyc1uiState,
+        submitData = { fname, lname,address1, address2,
+                       phoneno, dob ->
+            viewModel.submitData(
+                fname.trim { it <= ' ' },
+                lname.trim { it <= ' ' },
+                address1.trim { it <= ' ' },
+                address2.trim { it <= ' ' },
+                phoneno.trim { it <= ' ' },
+                dob.trim { it <= ' ' }
+            )
+        },
+        navigateBack = navigateBack
+    )
+}
+
+
+@Composable
+fun KYCLevel1Screen(
+    context:Context,
+    uiState: KYCLevel1UiState,
+    submitData: (String, String,String,String,String,String) -> Unit,
+    navigateBack:() -> Unit
+) {
+
+    when(uiState){
+        KYCLevel1UiState.EmptyForm -> {
+            Kyc1Form(
+                submitData = submitData
+            )
+        }
+
+        KYCLevel1UiState.Error -> {
+            Toast.makeText(context, stringResource(R.string.error_adding_KYC_Level_1_details_),Toast.LENGTH_SHORT).show()
+            navigateBack.invoke()
+        }
+
+        KYCLevel1UiState.Success -> {
+            Toast.makeText(context, stringResource(R.string.kyc_Level_1_details_added_successfully_),Toast.LENGTH_SHORT).show()
+            navigateBack.invoke()
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun Kyc1Form(
+    submitData: (String, String,String,String,String,String) -> Unit
+){
+
+    var firstName by remember { mutableStateOf("") }
+    var lastName by remember { mutableStateOf("") }
+    var address1 by remember { mutableStateOf("") }
+    var address2 by remember { mutableStateOf("") }
+    var mobileNumber by remember { mutableStateOf("") }
+    var dateOfBirth by remember { mutableStateOf("") }
+    val dateState = rememberUseCaseState()
+    val dateFormatter = DateTimeFormatter.ofPattern(stringResource(R.string.date_format))
+
+    Column(
+        modifier = Modifier
+            .padding(16.dp)
+            .fillMaxWidth()
+    ) {
+        OutlinedTextField(
+            value = firstName,
+            onValueChange = {
+                firstName = it },
+            label = {
+                Text(stringResource(R.string.first_name)) },
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 7.dp)
```

Comment:
we already have `MfOutlinedTextField ` and `MifosOutlinedTextField` so resuse them